x-pack/plugins/snapshot_restore/public/app/components/data_placeholder.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+interface Props {
+  data: any;
+  children: any,
```

Comment:
this could be typed as `React.ReactNode`

Reply:
Thanks! Done.